EventHandler.go

```diff
@@ -0,0 +1,42 @@
+package main 
```

Comment:
Maybe it would make sense to move this to a separate package. Root (`main`) package should be only for the `main.go` file which should be kept short. The `service` package is probably a good place.

As a side note, I'm guilty of not doing that myself when I started the project. In the meantime, I moved some (but not all) to separate packages.